lib/ansible/plugins/inventory/netbox.py

```diff
@@ -198,6 +198,8 @@ def group_extractors(self):
             "device_roles": self.extract_device_role,
             "platforms": self.extract_platform,
             "device_types": self.extract_device_type,
+            "config_context": self.extract_config_context,
```

Comment:
This is the change proposed by me at https://github.com/ansible/ansible/pull/47394 which hasn't been merged yet. Why have you included it here?